Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -65,7 +65,6 @@ variable (α : Type*) [Fintype α] [DecidableEq α]
 def alternatingGroup : Subgroup (Perm α) :=
   sign.ker
 
--- Porting note (https://github.com/leanprover-community/mathlib4/issues/10754): manually added instance
```

Comment:
As far as I can tell, this issue is still open: is there a reason to stop tracking it?

Reply:
I am proposing to declare almost all of these wontfix.

Reply:
I agree with Johan here: needing to copy instances for non-reducible `def`s is how Lean 4 is designed. Perhaps `deriving` should be improved so we don't have to do this manually, but the fact of copying is a wontfix.

Reply:
Makes sense. I agree deriving would be nice here too!